atlasdb-cassandra-multinode-tests/src/test/java/com/palantir/cassandra/multinode/OneNodeDownTableManipulationTest.java

```diff
@@ -57,21 +57,15 @@ public void canCreateTables() {
     }
 
     @Test
-    public void canDropTable() {
-        getTestKvs().dropTable(TABLE_TO_DROP);
-
-        assertThat(getTestKvs().getAllTableNames()).doesNotContain(TABLE_TO_DROP);
-        assertKvsReturnsEmptyMetadata(TABLE_TO_DROP);
-        assertCassandraSchemaChanged();
+    public void cannotDropTable() {
+        assertThrowsAtlasDbDependencyExceptionAndDoesNotChangeCassandraSchema(() ->
+                getTestKvs().truncateTable(TABLE_TO_DROP));
     }
 
     @Test
-    public void canDropTables() {
-        getTestKvs().dropTables(ImmutableSet.of(TABLE_TO_DROP_2));
-
-        assertThat(getTestKvs().getAllTableNames()).doesNotContain(TABLE_TO_DROP_2);
-        assertKvsReturnsEmptyMetadata(TABLE_TO_DROP_2);
-        assertCassandraSchemaChanged();
+    public void cannotDropTables() {
+        assertThrowsAtlasDbDependencyExceptionAndDoesNotChangeCassandraSchema(() ->
+                getTestKvs().truncateTable(TABLE_TO_DROP_2));
```

Comment:
Is this intentional? It seems like these tests should still call drop directly - especially since there are tables to be truncated in other tests.